chains/solana/contracts/programs/ccip-router/src/instructions/v1/messages.rs

```diff
@@ -0,0 +1,58 @@
+use anchor_lang::prelude::*;
```

Comment:
It looks like there is a typo in the file name, right? This should be something related to pools

Reply:
Well, I extracted these from the `messages.rs` so I kept the name. I get that they are related to pools, so I'm happy to move them there if you think that's better, but maintaining the name was the "safer" refactor given that I haven't dived too deep into how those structs are used